<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Main</class>
 <widget class="QMainWindow" name="F_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Калькулятор коктейлей</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Recources.qrc">
    <normaloff>:/Icons/Icons/Cocktail_Icon.png</normaloff>:/Icons/Icons/Cocktail_Icon.png</iconset>
  </property>
  <widget class="QWidget" name="Main_Widget">
   <layout class="QGridLayout" name="Main_Layout">
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="Add_Ingredient_Layout">
      <item>
       <widget class="QPushButton" name="Add_Ingredient_Button">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Recources.qrc">
          <normaloff>:/Icons/Icons/Add_Icon.png</normaloff>:/Icons/Icons/Add_Icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="Cocktails_Layout">
      <item>
       <widget class="QLabel" name="Cocktails_Label">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Коктейли</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="Add_Cocktail_Layout">
      <item>
       <widget class="QPushButton" name="Add_Cocktail_Button">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Recources.qrc">
          <normaloff>:/Icons/Icons/Add_Icon.png</normaloff>:/Icons/Icons/Add_Icon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="Ingredients_Layout">
      <item>
       <widget class="QLabel" name="Ingredients_Label">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Ингредиенты</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QScrollArea" name="Ingredients_Scroll_Area">
      <property name="minimumSize">
       <size>
        <width>333</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>333</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="Ingredients_Scroll_Area_Content">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>331</width>
         <height>484</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QScrollArea" name="Cocktails_Scroll_Area">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="Cocktails_Scroll_Area_Content">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>641</width>
         <height>484</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Recources.qrc"/>
 </resources>
 <connections/>
</ui>
